front/src/modules/users/hooks/useOpenUserSearch.ts

```diff
@@ -0,0 +1,10 @@
+import { RelationPickerHotkeyScope } from '@/ui/input/relation-picker/types/RelationPickerHotkeyScope';
+import { useSetHotkeyScope } from '@/ui/utilities/hotkey/hooks/useSetHotkeyScope';
+
+export function useOpenUserSearch() {
```

Comment:
The naming is misleading here, it sets the hotkey scope to relation picker, and it does not open anything. 

Keeping the `setHotkeyScope(RelationPickerHotkeyScope.RelationPicker);` in the useEffect directly is fine.